### PR TITLE
network: VXlan group and remote fixes

### PR DESCRIPTION
**Pull request by ssahani**
closes https://github.com/systemd/systemd/issues/15391

### CODE REVIEW

**Comment by keszybz** [comment]
Looks superficially OK, but a review from someone who groks the logic here would be good. @yuwata ?

**Comment by fooker** [comment]
I can confirm that the fix solves the problem as described in #15391.

**Comment by fooker** [comment]
Thanks for the quick fix.

**Comment by poettering** [comment]
lgtm, too. Given the positive feedback, let's merge. a review post-merge from @yuwata would still be great of course.

File: src/network/netdev/vxlan.c [hunk 1]
```diff
@@ -38,7 +38,14 @@ static int netdev_vxlan_fill_message_create(NetDev *netdev, Link *link, sd_netli
...
 
-        if (in_addr_is_null(v->remote_family, &v->remote) == 0) {
+        if (in_addr_is_null(v->group_family, &v->group) == 0) {
+                if (v->group_family == AF_INET)
+                        r = sd_netlink_message_append_in_addr(m, IFLA_VXLAN_GROUP, &v->group.in);
+                else
+                        r = sd_netlink_message_append_in6_addr(m, IFLA_VXLAN_GROUP6, &v->group.in6);
+                if (r < 0)
+                        return log_netdev_error_errno(netdev, r, "Could not append IFLA_VXLAN_GROUP attribute: %m");
+        } else  if (in_addr_is_null(v->remote_family, &v->remote) == 0) {
```
Comment (yuwata) [inline src/network/netdev/vxlan.c:48]: redundant space.

**Pull request merged**